### ISSUE TITLE
Mechanism for reporting a11y issues

### ISSUE THREAD

**Issue by u9000-Nine** [w3c/silver#708]
In tuesday's meeting, it was mentioned that having a mechanism for reporting accessibility issues would be a good idea. I said that it would be best to make sure that this mechanism is itself accessible, so that people who are impacted by accessibility issues can actually report them. I'm thinking something along the lines of `security.txt` but for accessibility (`a11y.txt`?).

Per the meeting, I have added this as an issue. I also searched the issues in this and the WCAG repositories, and have not found anything that could be considered a duplicate. If this is the wrong place for this issue, please let me know and I will fix it.

**Comment by u9000-Nine** [comment]
I mean that there should be a recommendation that sites should provide this feature.

Apologies for the late response; I somehow just noticed yours.

**Comment by Myndex** [comment]
### Something like:

    /accessibility.txt

### With contents such as

```txt
## Accessibility Metadata for example.com

Canonical: https://example.com/accessibility.txt           ## This file's URL

## ExampleCo, Inc. Accessibility Dept. Contact Information

Contact: mailto:a11yadmin@example.com                      ## Accessibility dept. contact
Contact: tel:+1-213-555-5555                               ## Accessibility dept. phone
Contact: fax:+1-213-555-1234                               ## Accessibility dept. fax
Contact: rtty:+1-213-555-1337                              ## Accessibility dept. teletype
Contact: Address:                                          ## Accessibility dept. postal address
                  ExampleCo, Inc.
                  P.O.Box 1234
                  ATTN: Accessibility officer
                  Springville, NY, 10000
Support: https://example.com/chat-support#a11y             ## web based chat support
Comment: https://example.com/feedback#a11y                 ## web based feedback form
Surveys: https://example.com/accessibility-survey          ## web based user survey

## ExampleCo, Inc., Accessibility Statement, Legal Policies, Etc.

Statement: https://example.com/a11y-statement              ## Accessibility statement
Legal: https://example.com/legal#a11y                      ## Legal terms of service
Hiring: https://example.com/jobs#a11y                      ## Positions available
Acknowledgments: https://example.com/thanks#a11y           ## Contributors
WAI: https://example.com/.well-known/wai/wai-metadata.json ## WAI JSON file

Preferred-Languages: en, en-us, en-uk, fr, ru              ## Supported language(s)

LastUpdate: 2023-08-29T21:00:00.000Z                       ## This file's modified date

```
